### ISSUE TITLE
Problem with manual fan control.

### ISSUE THREAD

**Issue by Autobiography59** [seerge/g-helper#1337]
Hello, firstly, I would like to thank you for the excellent application, secondly, my knowledge of English is not perfect, please forgive me in advance. I have an Asus Scar 16 laptop (2023). Manual fan control settings are not applied, if i click the check mark, they start spinning at maximum speed.

[log.txt](https://github.com/seerge/g-helper/files/12677339/log.txt)

**Comment by seerge** [comment]
@Autobiography59 hello,

From log i see that curve is being sent to BIOS, and it's not max fans, it's normal

```
21.09.2023 2:05:39: FanCPU = 1E-32-3E-42-4A-50-52-56-00-01-04-09-0E-11-2B-33 : OK
21.09.2023 2:05:39: FanGPU = 1E-32-3D-42-4B-4E-52-56-00-01-07-0C-16-28-30-3D : OK
21.09.2023 2:05:39: FanMid = 1E-32-3E-42-4B-4E-52-56-00-01-06-0B-10-23-28-2B : OK
```

What is maximum speed in RPMs ? and what are the temperatures on that moment?

There is nothing else app can do that, next to this :) as method to set fans is same for all asus laptops

Try to set custom fan curves for different modes (not for balanced, but for turbo for example), reboot, reinstall Asus System Control interface or do a [hardware reset](https://github.com/seerge/g-helper#how-do-i-do-a-hardware-reset-on-a-laptop)

**Comment by Autobiography59** [comment]
I changed the fan settings to a custom profile. before that, this profile was at the maximum position of the fans, perhaps he remembered it once and did not want to change it anymore. Now I have deleted that profile and added a new one, everything seems to be working. At some point everything started working, I do not know what happened even (after I wrote to you). Before that, on standard presets, too, nothing changed when the checkmark was clicked. The last thing you looked at in the log, I tried to change it already in the standard preset (nothing happened-no reaction). When I did this in what I created myself(before i wrote to you), the fans just turned at maximum (6900 rpm +)
